### DISCUSSION TITLE
AlDente 1.23 Pappardelle

### DISCUSSION THREAD

**Discussion by davidwernhart** [AppHouseKitchen/AlDente-Battery_Care_and_Monitoring#1092] [General]
- [All] Added support for the new M3 based MacBook Pros.
- [Pro] The current battery percentage is now also shown as text in the status bar.
- [Pro] The calibration state is now shown in the Popup while a calibration is running.
- [Pro] Slight status bar design improvements.
- [Pro] Fixed a bug that caused MacBook to always charge to 100% when AlDente is closed in Intel Mode.
- [Pro] Fixed a bug that caused the "Control Magsafe LED" Option to not appear in the settings for Thunderbolt 2 MacBooks.
- [Pro] Fixed a bug that caused "Disable Sleep until Charge Limit" to not work correctly in some scenarios.

With peace & love from Vienna,  
AppHouseKitchen

<hr /><em>This discussion was created from the release <a href='https://github.com/AppHouseKitchen/AlDente-Charge-Limiter/releases/tag/1.23'>AlDente 1.23 Pappardelle</a>.</em>

**Comment by Gavinyul96** [comment]
是否可以在官网上也同步软件更新日志呢？一个小小的建议

**Comment by edward-bailey** [comment]
Thankyou, as a new Mac user i was going slightly crazy with the Intel Mode not working and always going to 100%, I'll give this variant a go. (Note I'm using the free version to trial before deciding if i should upgrade)

**Reply by MatthiasKerbl** [reply]
Hi @edward-bailey,

Unfortunately, Intel Mode is still inconsistent in AlDente 1.23. However, it will be fixed in the next update.

Best,
Matthias
